1_basics.ipynb

```diff
@@ -254,14 +502,28 @@
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "lang": "ja"
+   },
    "source": [
     "<div class=\"alert alert-block alert-info\">\n",
     "    <b>次の学習</b>\n",
     "    <p>HSRに搭載されているセンサの情報を取得し、台車を制御して壁の手前まで動かしてみます。</p>\n",
     "</div>"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "lang": "en"
+   },
+   "source": [
+    "<div class=\"alert alert-block alert-info\">\n",
+    "    <b>What's next</b>\n",
+    "    <p>In the next notebook, we will capture the information from the sensors mounted on HSR, and move the mobile base until it reach to the front of the wall.</p>\n",
```

Comment:
"    <p>In the next notebook, we will capture the information from the sensors mounted on **the** HSR, and move the mobile base until it **reaches the front of the wall.**</p>\n",

Added "the" in front of "HSR" and updated verb agreement with "it" and "reach" as well as deleted the "to" after the verb "reach".